docs/eventing/sources/apiserversource/getting-started/_index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Getting started"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/samples/kubernetes-event-source
+---
+
+## Prerequisites
+
+Before you can create an API server source, you must install Knative Eventing and the `kubectl` CLI tool.
+
+## Create an API server source
+
+1. Optional: Create a namespace for the API server source instance:
+
+    ```shell
+    kubectl create namespace <new_namespace_name>
+    ```
+
+    Creating a namespace for your API server source and related components allows you to view changes and events for this workflow more easily, since these are isolated from the many other components that may exist in your `default` namespace.
+
+    It also makes removing the source easier, since you can simply delete the namespace to remove all of the resources.
+
+1. Create a service account:
+
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: <service_account_name>
+      namespace: <your_namespace>
+    ```
+
+1. Create a cluster role:
+
+    ```yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: <cluster_role_name>
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - events
+      verbs:
+      - get
+      - list
+      - watch
+    ```
+
+1. Create a cluster role binding:
+
+    ```yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: <cluster_role_binding_name>
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: <your_cluster_role>
```

Comment:
Also the Google style guide topic I linked above discourages using possessive pronouns in placeholder variables. So instead of `<your_cluster_role>` Google recommends `CLUSTER_ROLE_NAME`.

Reply:
that seems acceptable, I just wasn't sure what to call it

Reply:
I'm going to go with <this-format> as per https://kubernetes.io/docs/contribute/style/style-guide/#use-angle-brackets-for-placeholders but I will update the names 🙂